run_cv.py

```diff
@@ -47,10 +49,18 @@
     target_detector = LoadingBayDetector(generator)
 elif args.target == "power_port":
     target_detector = PowerPortDetector(generator)
+    pc = PoseCalculator(target_detector)
+elif args.target == "power_cell":
+    target_detector = PowerCellDetector(generator)
+    pc = BallFinder(target_detector)
 
-with PoseCalculator(target_detector) as pc:
-    while (generator.is_capturing() if args.generator == "video_file" else True):
+while generator.is_capturing() if args.generator == "video_file" else True:
+    if type(target_detector) is PowerCellDetector:
```

Comment:
Test the value of args.target, calling type() on objects is frowned upon for Python.